Source/Csla/Server/DefaultExceptionInspector.cs

```diff
@@ -0,0 +1,30 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="DefaultExceptionInspector.cs" company="Marimer LLC">
+//     Copyright (c) Marimer LLC. All rights reserved.
+//     Website: https://cslanet.com
+// </copyright>
+// <summary>Default implementation of exception inspector.</summary>
+//-----------------------------------------------------------------------
+using System;
+
+namespace Csla.Server
+{
+  /// <summary>
+  /// Default implementation of exception inspector.
+  /// </summary>
+  internal class DefaultExceptionInspector : IDataPortalExceptionInspector
```

Comment:
I would like to create a sanitizing exception inspector too - but I can do that as a separate task.